outport/notifier/eventNotifier.go

```diff
@@ -46,10 +48,15 @@ func NewEventNotifier(args ArgsEventNotifier) (*eventNotifier, error) {
 	if err != nil {
 		return nil, err
 	}
+	blockContainer, err := createBlockCreatorsContainer()
```

Comment:
Please move it at the same level as factory: `CreateEventNotifier`, just like you did on indexer

Reply:
done